gammapy/estimators/map/asmooth.py

```diff
@@ -39,9 +39,9 @@ class ASmoothMapEstimator(Estimator):
     spectrum : `SpectralModel`
         Spectral model assumption.
     method : {'asmooth', 'lima'}
```

Comment:
```suggestion
    method : {'lima', 'asmooth'}
```
Typical syntax is that the first entry in the list is the default.